content/applications/essentials/export_import_data.rst

```diff
@@ -2,166 +2,245 @@
 Export and import data
 ======================
 
+In Odoo, it is sometimes necessary to export or import data for running reports, or for data
+modification. This document covers the export and import of data into and out of Odoo.
+
+.. important::
+   Sometimes, users run into a 'time out' error, or a record does not process, due to its size. This
+   can occur with large exports, or in cases where the import file is too large. To circumvent this
+   limitation surrounding the size of the records, process exports or imports in smaller batches.
+
 .. _export-data:
 
 Export data from Odoo
 =====================
 
-When working with a database, it sometimes is necessary to export your data in a distinct file.
-Doing so can help to do reporting over your activities (even if Odoo offers a precise and easy
-reporting tool with each available application).
+When working with a database, it is sometimes necessary to export data in a distinct file. Doing so
+can aid in reporting on activities, although, Odoo provides a precise and easy reporting tool with
+each available application.
 
-With Odoo, you can export the values from any field in any record. To do so,
-activate the list view on the items that need to be exported, click on *Action*, and, then,
-on *Export*.
+With Odoo, the values can be exported from any field in any record. To do so, activate the list view
+(:guilabel:`≣ (four horizontal lines)` icon), on the items that need to be exported, and then select
```

Comment:
Sorry my suggestion was not accurate, in 15.0 the list icon view has three lines instead of 4 🙃
```suggestion
(:guilabel:`☰ (three horizontal lines)` icon), on the items that need to be exported, and then select
```